openshift/secret-template.yaml

```diff
@@ -39,6 +39,6 @@ objects:
       labels:
         app: srcopsmetrics
     stringData:
-      githubAccessToken: ${GITHUB_ACCESS_TOKEN}
-      thothCephKeyID: ${THOTH_CEPH_KEY_ID}
-      thothCephSecretKey: ${THOTH_CEPH_SECRET_KEY}
+      github-access-token: ${GITHUB_ACCESS_TOKEN}
+      ceph-key-id: ${CEPH_KEY_ID}
+      ceph-secret-key: ${CEPH_SECRET_KEY}
```

Comment:
so we are using kebab case for the envs?
is somewhere a doc that would describe our standards on openshift? - so i know for the future

Reply:
We can maybe write one :) @fridex WDTY? I just follow our ConfigMap like Thoth one.

Reply:
Yeah, we don't have any guide on this. We can do that, I just follow the rest of the code most of the time :)